src/components/CommonModal/index.module.scss

```diff
@@ -0,0 +1,27 @@
+.overlay {
+  display: block;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100vw;
```

Comment:
`width` should be `100%` because `100vw` includes the width of scrollbar, and offset will be there if the scrollbar is visible.

Ref: https://stackoverflow.com/questions/23367345/100vw-causing-horizontal-overflow-but-only-if-more-than-one